cinch/roles/jenkins_slave/templates/sysconfig_jenkins_swarm

```diff
@@ -6,4 +10,4 @@ SWARM_SLAVE_LABEL="{{ jslave_label }}"
 SWARM_ROOT="{{ jswarm_home }}"
 SWARM_USERNAME="{{ (jenkins_slave_username == '') | ternary('', '-username') }} {{ jenkins_slave_username }}"
 SWARM_PASSWORD="{{ (jenkins_slave_password == '') | ternary('', '-password') }} {{ jenkins_slave_password }}"
-SWARM_EXTRA_ARGS="-mode exclusive -disableSslVerification -deleteExistingClients"
+SWARM_EXTRA_ARGS="-mode exclusive -disableSslVerification -deleteExistingClients {{ jswarm_extra_args }}"
```

Comment:
What limitation prevents us from sharing the generated swarm CLI between the three places that need it? Just curious.